src/Contracts/User.php

```diff
@@ -38,4 +38,11 @@ public function getEmail();
      * @return string
      */
     public function getAvatar();
+
+    /**
+     * Get the gender of the user.
+     *
+     * @return string
+     */
+    public function getGender();
```

Comment:
Absolutely not. I just edited socialite in project to pull gender and thought other people may need this function too.